src/Microsoft.ML.CodeGenerator/Templates/AzureImageClassification/Model/NormalizeMapping.cs

```diff
@@ -0,0 +1,356 @@
+                    " operation.\r\n        public override Action<NormalizeInput, NormalizeOutput> Get" +
+                    "Mapping()\r\n        {\r\n            return Mapping;\r\n        }\r\n    }\r\n    public " +
+                    "class NormalizeInput\r\n    {\r\n        [ColumnName(\"ImageSource_featurized\")]\r\n   " +
+                    "     [VectorType(3, 224, 224)]\r\n        public VBuffer<float> Reshape;\r\n    }\r\n " +
+                    "   public class NormalizeOutput\r\n    {\r\n        [ColumnName(\"input1\")]\r\n        " +
+                    "[VectorType(3 * 224 * 224)]\r\n        public VBuffer<float> Reshape;\r\n    }\r\n}\r\n\r" +
+                    "\n");
+            return this.GenerationEnvironment.ToString();
+        }
+
+public string Namespace {get;set;}
+internal CSharp.GenerateTarget Target {get;set;}
+
+
+void CLI_Annotation()
+{
+this.Write(@"//*****************************************************************************************
+//*                                                                                       *
+//* This is an auto-generated file by Microsoft ML.NET CLI (Command-Line Interface) tool. *
+//*                                                                                       *
+//*****************************************************************************************
+");
+
+
+}
+
+
+void MB_Annotation()
+{
+this.Write("// This file was auto-generated by ML.NET Model Builder. \r\n");
+
+
+}
+
+    }
+    #region Base class
+    /// <summary>
+    /// Base class for this transformation
+    /// </summary>
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "16.0.0.0")]
+    internal class NormalizeMappingBase
+    {
+        #region Fields
+        private global::System.Text.StringBuilder generationEnvironmentField;
+        private global::System.CodeDom.Compiler.CompilerErrorCollection errorsField;
+        private global::System.Collections.Generic.List<int> indentLengthsField;
+        private string currentIndentField = "";
+        private bool endsWithNewline;
+        private global::System.Collections.Generic.IDictionary<string, object> sessionField;
+        #endregion
+        #region Properties
+        /// <summary>
+        /// The string builder that generation-time code is using to assemble generated output
+        /// </summary>
+        protected System.Text.StringBuilder GenerationEnvironment
+        {
+            get
+            {
+                if ((this.generationEnvironmentField == null))
+                {
+                    this.generationEnvironmentField = new global::System.Text.StringBuilder();
+                }
+                return this.generationEnvironmentField;
+            }
+            set
+            {
+                this.generationEnvironmentField = value;
+            }
+        }
+        /// <summary>
```

Comment:
This whole file is missing white space between code blocks.

Reply:
because It's auto-generated by t4

Reply:
## Resolved